scripts/RedsHouse2F.asm

```diff
@@ -22,5 +22,5 @@ RedsHouse2FNoopScript:
 	ret
 
 RedsHouse2F_TextPointers:
-
+	def_text_pointers
```

Comment:
For these 4 files, I would leave the blank line after the `def_text_pointers` since the unused `text_end` is unrelated and not a list terminator or anything